src/models/streaming/BufferedDuplex.ts

```diff
@@ -0,0 +1,43 @@
+import { Duplex, PassThrough } from 'stream';
+
+export default class BufferedDuplex extends Duplex {
+    input: PassThrough;
+    output: PassThrough;
+    areOutputHandlersSetup: boolean;
+    constructor (input: PassThrough, output: PassThrough, options?: any) {
```

Comment:
does this `options` value have a type?

Reply:
https://nodejs.org/api/stream.html#stream_new_stream_transform_options It does not according to this documentation